fhir-parent/pom.xml

```diff
@@ -106,6 +112,8 @@
         <module>../operation/fhir-operation-bulkdata</module>
         <module>../operation/fhir-operation-everything</module>
         <module>../operation/fhir-operation-erase</module>
+        <module>../operation/fhir-operation-cpg</module>
+        <module>../operation/fhir-operation-cqf</module>
```

Comment:
Let's put these modules under cql/operation cql/ you can create a pom module in cql/ too, but not required